descriptors/ModuleDescriptor-template.json

```diff
@@ -208,7 +208,7 @@
           ],
           "pathPattern":"/accounts/{id}/check-pay",
           "permissionsRequired":[
-            "accounts.check-pay.post"
+            "feesfines.accounts.check-pay.item.post"
```

Comment:
This endpoint represents an action performed on "account"(s). And these actions are not "items", not really. So a more appropriate permission name in this case would be `feesfines.accounts.check-pay.post` or `feesfines.accounts.check-pay.execute`. Same applies to all other action-related permissions below.

Reply:
Thanks, fixed